packages/driver/src/cross-origin/patches/cookies.js

```diff
@@ -1,3 +1,46 @@
+const parseCookieString = (cookieString) => {
```

Comment:
Is this a replacement for the tough cookie parsing or are they parsing different cookie string formats?

Reply:
Essentially they're different. Tough cookie will only parse a single cookie string, namely the one in `Set-Cookie` response headers. This is for parsing the `document.cookie` string, which includes multiple cookies in the format `key1=value1; key2=value2`.